config/packages/messenger.yaml

```diff
@@ -0,0 +1,18 @@
+framework:
+    messenger:
+        # Uncomment this (and the failed transport below) to send failed messages to this transport for later handling.
+        # failure_transport: failed
```

Comment:
I think we should enable a failure transport on all services to be able to monitor and debug errors